main/src/cgeo/geocaching/network/Network.java

```diff
@@ -57,40 +50,43 @@
 
     private static final Pattern PATTERN_PASSWORD = Pattern.compile("(?<=[\\?&])[Pp]ass(w(or)?d)?=[^&#$]+");
 
-    private final static HttpParams CLIENT_PARAMS = new BasicHttpParams();
-
-    static {
-        CLIENT_PARAMS.setParameter(CoreProtocolPNames.HTTP_CONTENT_CHARSET, CharEncoding.UTF_8);
-        CLIENT_PARAMS.setParameter(CoreConnectionPNames.CONNECTION_TIMEOUT, 30000);
-        CLIENT_PARAMS.setParameter(CoreConnectionPNames.SO_TIMEOUT, 30000);
-        CLIENT_PARAMS.setParameter(ClientPNames.HANDLE_REDIRECTS, true);
-    }
+    private final static OkHttpClient OK_HTTP_CLIENT = new OkHttpClient.Builder()
+            .connectTimeout(30, TimeUnit.SECONDS)
```

Comment:
since the configuration has different settings for read and write timeout here, would it make sense to think about different values? e.g. have a higher value for write, assuming it is more important to save user input (logs, field notes, ...).